ChatTTS/core.py

```diff
@@ -519,65 +527,57 @@ def _infer_code(
             )
             input_ids = [i.tolist() for i in input_ids]
 
-            result = gpt.llm.generate(
-                None,
-                sample_params,
-                input_ids,
+            results_generator = gpt.llm.llm_engine.generate(
+                None, sample_params, uuid.uuid4(), speaker_embedding_param, input_ids[0]
             )
-
-            token_ids = []
-            hidden_states = []
-            for i in result:
-                token_ids.append(torch.tensor(i.outputs[0].token_ids))
-                hidden_states.append(
-                    i.outputs[0].hidden_states.to(torch.float32).to(self.device)
-                )
-
-            del text_mask, input_ids
-
-            return [
-                GPT.GenerationOutputs(
-                    ids=token_ids,
-                    hiddens=hidden_states,
-                    attentions=[],
-                ),
-            ]
-
-        emb = self.embed(input_ids, text_mask)
```

Comment:
If you move a line up, don't change the variable name when it's ok to remain the original one. It will make it difficult to see your changes.